service.go

```diff
@@ -21,7 +23,9 @@ type Service interface {
 	ServiceName() string
 	TrackName() string
 	URLRegex(string) bool
+	SearchRegex(string) bool
```

Comment:
This function doesn't seem necessary to me since it is just performing a single boolean statement.